pypeit/specobjs.py

```diff
@@ -621,7 +622,8 @@ def apply_flux_calib(self, par, spectrograph, sens):
                                              longitude=spectrograph.telescope['longitude'],
                                              latitude=spectrograph.telescope['latitude'],
                                              extinctfilepar=par['extinct_file'],
-                                             airmass=float(self.header['AIRMASS']))
+                                             airmass=float(self.header['AIRMASS']),
+                                             sens_fwhm = sens.sens['SENS_STD_SPAT_FWHM'][indx[0]])
```

Comment:
Let's remove the sens_fwhm options until they are actually used.

Reply:
removed.